src/components/Button/Button.tsx

```diff
@@ -16,21 +18,23 @@ export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
 
 export interface StyledButtonProps {
   styleType: ButtonType;
+  disabled?: boolean;
   align?: Alignment;
   width?: string;
   height?: string;
 }
 
 export const Button = ({
   type = "primary",
+  disabled = false,
```

Comment:
we can remove the disabled property from here, it will be taken care of in the `delegated`